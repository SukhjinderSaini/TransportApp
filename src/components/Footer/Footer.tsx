import React from "react";
import { Navbar, Container, Nav } from "react-bootstrap";
function Footer() {
  return (
    <Navbar fixed="bottom" expand="md" bg="dark" data-bs-theme="dark">
      <Container className="p-2 pe-3 text-light" fluid>
        <Navbar.Toggle aria-controls="basic-navbar-nav" />
        <Navbar.Collapse
          className="justify-content-center"
          id="responsive-navbar-nav">
          <Nav.Link href="#action1" className="px-3 px-md-3">
            Home
          </Nav.Link>
          <span className="align-self-center d-none d-md-inline px-md-2">
            |
          </span>
          <Nav.Link href="#action2" className="px-2 px-sm-3 px-md-3">
            About
          </Nav.Link>
          <span className="align-self-center d-none d-md-inline px-md-2">
            |
          </span>
          <Nav.Link href="#action3" className="px-2 px-sm-3 px-md-3">
            Copyright
          </Nav.Link>
          <span className="align-self-center d-none d-md-inline px-md-2">
            |
          </span>
          <Nav.Link href="#action4" className="px-2 px-sm-3 px-md-3">
            Services
          </Nav.Link>
          <span className="align-self-center d-none d-md-inline px-md-2">
            |
          </span>
          <Nav.Link href="#action5" className="px-2 px-sm-3 px-md-3">
            Contact Us
          </Nav.Link>
        </Navbar.Collapse>
      </Container>
    </Navbar>
  );
}

export default Footer;
