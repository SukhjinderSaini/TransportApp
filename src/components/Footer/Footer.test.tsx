import React from "react";
import renderer from "react-test-renderer";

import Footer from "./Footer";
it("renders footer correctly", () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
