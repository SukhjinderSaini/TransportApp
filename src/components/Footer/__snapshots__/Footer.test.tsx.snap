// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<nav
  className="navbar navbar-expand-md navbar-light bg-dark fixed-bottom"
  data-bs-theme="dark"
>
  <div
    className="p-2 pe-3 text-light container-fluid"
  >
    <button
      aria-controls="basic-navbar-nav"
      aria-label="Toggle navigation"
      className="navbar-toggler collapsed"
      onClick={[Function]}
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      aria-expanded={null}
      className="justify-content-center navbar-collapse collapse"
      id="responsive-navbar-nav"
    >
      <a
        className="px-3 px-md-3 nav-link"
        disabled={false}
        href="#action1"
        onClick={[Function]}
      >
        Home
      </a>
      <span
        className="align-self-center d-none d-md-inline px-md-2"
      >
        |
      </span>
      <a
        className="px-2 px-sm-3 px-md-3 nav-link"
        disabled={false}
        href="#action2"
        onClick={[Function]}
      >
        About
      </a>
      <span
        className="align-self-center d-none d-md-inline px-md-2"
      >
        |
      </span>
      <a
        className="px-2 px-sm-3 px-md-3 nav-link"
        disabled={false}
        href="#action3"
        onClick={[Function]}
      >
        Copyright
      </a>
      <span
        className="align-self-center d-none d-md-inline px-md-2"
      >
        |
      </span>
      <a
        className="px-2 px-sm-3 px-md-3 nav-link"
        disabled={false}
        href="#action4"
        onClick={[Function]}
      >
        Services
      </a>
      <span
        className="align-self-center d-none d-md-inline px-md-2"
      >
        |
      </span>
      <a
        className="px-2 px-sm-3 px-md-3 nav-link"
        disabled={false}
        href="#action5"
        onClick={[Function]}
      >
        Contact Us
      </a>
    </div>
  </div>
</nav>
`;

exports[`renders footer correctly 1`] = `
<nav
  className="navbar navbar-expand-md navbar-light bg-dark fixed-bottom"
  data-bs-theme="dark"
>
  <div
    className="p-2 pe-3 text-light container-fluid"
  >
    <button
      aria-controls="basic-navbar-nav"
      aria-label="Toggle navigation"
      className="navbar-toggler collapsed"
      onClick={[Function]}
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      aria-expanded={null}
      className="justify-content-center navbar-collapse collapse"
      id="responsive-navbar-nav"
    >
      <a
        className="px-3 px-md-3 nav-link"
        disabled={false}
        href="#action1"
        onClick={[Function]}
      >
        Home
      </a>
      <span
        className="align-self-center d-none d-md-inline px-md-2"
      >
        |
      </span>
      <a
        className="px-2 px-sm-3 px-md-3 nav-link"
        disabled={false}
        href="#action2"
        onClick={[Function]}
      >
        About
      </a>
      <span
        className="align-self-center d-none d-md-inline px-md-2"
      >
        |
      </span>
      <a
        className="px-2 px-sm-3 px-md-3 nav-link"
        disabled={false}
        href="#action3"
        onClick={[Function]}
      >
        Copyright
      </a>
      <span
        className="align-self-center d-none d-md-inline px-md-2"
      >
        |
      </span>
      <a
        className="px-2 px-sm-3 px-md-3 nav-link"
        disabled={false}
        href="#action4"
        onClick={[Function]}
      >
        Services
      </a>
      <span
        className="align-self-center d-none d-md-inline px-md-2"
      >
        |
      </span>
      <a
        className="px-2 px-sm-3 px-md-3 nav-link"
        disabled={false}
        href="#action5"
        onClick={[Function]}
      >
        Contact Us
      </a>
    </div>
  </div>
</nav>
`;
